### DISCUSSION TITLE
Website Improvements

### DISCUSSION THREAD

**Discussion by zainfathoni** [kawalcovid19/wargabantuwarga.com#23] [Ideas]
## Feedbacks

1. Docs dan Sheets terpisah
  a. less awareness of the sheets existence
  b. unfamiliarity of searching by lokasi/kebutuhan
2. Information Architecture

## Action Items

- [x] UX Improvements (1)
  - [x] Manpower issues: designrant @zainfathoni
- [x] Analytics #18 (2) ETA pending @Bara
- [x] Refine Information Architecture (2)
- [x] CMS Migration #31 (4)

### About Information Architecture

The most important content types in our homepage are:

1. Hotline information
2. Database sheet https://kcov.id/wbw-sheets
3. How to contribute
4. Education section (Daftar Isi)

## Questions

- The added value of synchronizing with urundayacovid.com instead of just linking to the original website
  - no significant update value
  - we can utilise it as a funnel for collecting data

**Comment by baraeb92** [comment]
Sip, nanti kita ngasi updatesnya disini aja ya mas berarti. 

Kalau udah ada ETAnya nanti bisa share disini juga.

**Reply by zainfathoni** [reply]
Iya Mas @baraeb92.
Saya barusan juga udah kasih akses Mas untuk bantuin Triage issues di GitHub repository ini.
Idealnya, feedbacks & action items di atas kita _convert_ jadi issues supaya lebih mudah ngelola _milestones_-nya.

**Comment by josegautama** [comment]
UI & UX boleh di improve.
utk UI boleh dibuat spt web pada umum nya.

apabila kekurangan relawan, sy siap bantu

**Reply by baraeb92** [reply]
Boleh kasi feedbacknya nanti dithread ini ya mas, supaya centralized. 

https://github.com/kawalcovid19/wargabantuwarga.com/discussions/16#discussioncomment-991862
